src/specs/api.v2.yaml

```diff
@@ -188,6 +188,72 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/HTTPError'
+  '/experiments/{experimentId}/samples/{sampleId}':
+    post:
+      summary: Create sample
+      operationId: createSample
+      x-eov-operation-id: sample#createSample
+      x-eov-operation-handler: routes/sample
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CreateSample'
+      responses:
+        '200':
+          description: Success
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/HTTPSuccess'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/HTTPError'
+        '401':
+          description: The request lacks authentication credentials.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/HTTPError'
+        '404':
+          description: Not found error.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/HTTPError'
+    delete:
+      summary: Delete sample
+      operationId: deleteSample
+      x-eov-operation-id: sample#deleteSample
+      x-eov-operation-handler: routes/sample
+      responses:
+        '200':
+          description: Success
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/HTTPSuccess'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/HTTPError'
+        '401':
```

Comment:
If I am not mistaken, the API returns 403 for non-authorized errors, not 401. Can you double-check?

Reply:
401 is lacking authentication credentials, 403 is forbidden, so credentials exist but not permission

Reply:
I'll add 403 as well